src/content/docs/logs/ui-data/query-time-parsing.mdx

```diff
@@ -0,0 +1,419 @@
+
+import logsViewResultsinLogDetailsView from 'images/logs_screenshot-full_view-results-in-log-details-view.webp'
+
+import logsLogTableResults from 'images/logs_screenshot-full_log-table-results.webp'
+
+import logsRuleManager from 'images/logs_screenshot-crop_rule-manager.webp'
+
+import logsQueryTimeSearchBar from 'images/logs_screenshot-full_query-time-search-bar.webp'
+
+import logsAlertQueryBuilder from 'images/logs_screenshot-crop_alert-query-builder.webp'
+
+import logsQueryBehindParsingRule from 'images/logs_screenshot-crop_query-behind-parsing-rule.webp'
+
+import logsParsingAttributesView from 'images/logs_screenshot-crop_parsing-attributes-view.webp'
+
+
+
+
+Are you looking for a quick way to visually extract attributes from your logs after they've been ingested into New Relic? Query time parsing lets you parse your logs directly in the UI without needing to write complex regular expressions or Grok patterns. You can use query time parsing to temporarily extract values from your logs and quickly perform a query on these variables. The results are shown instantly since parsing is performed at query time. 
+
+## How query time parsing differs from ingest parsing [#differences]
+
+While both types of parsing make it easier for you to query logs, they have some significant differences:
+
+* **Ingest parsing:** Parsing during [log ingestion](/docs/logs/ui-data/parsing/) is where you create parsing rules using Grok or regular expressions (or both). As log records are ingested at New Relic, your parsing rules are applied to create permanent attributes that are stored with your log data in NRDB. These attributes make it easier for you to query log data. 
+
+* **Query time parsing:** In contrast to ingest parsing, query time parsing lets you create temporary attributes that will be used as query variables. You can then use these variables in NRQL queries to populate your log table. We automatically create the queries as you make selections in the UI. 
+
+You may also choose to use a combination of both parsing approaches. Review the table below to decide if query time parsing is right for you:
+
+<table>
+    <thead>
+        <tr>
+            <th>
+                Description 
+            </th>
+            <th>
+                Ingest parsing 
+            </th>
+            <th>
+                Query time parsing 
+            </th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>
+                Recommended usage
+            </td>
+            <td>
+                Best for creating permanent attributes you can query in the future
+            </td>
+            <td>
+                Best for doing quick queries on attributes that aren't permanent 
+            </td>
+        </tr>
+        <tr>
+            <td>
+                Parsing language                
+            </td>
+            <td>
+                You create grok patterns and regular expressions 
```

Comment:
Ah sorry I missed this instance of capitalizing Grok as well. I believe this is the last one.

Reply:
@mchavez-newrelic another good catch! I'll fix this!